vit/command_bar.py

```diff
@@ -20,15 +21,7 @@ def keypress(self, size, key):
         if key not in ('tab', 'shift tab'):
             self.autocomplete.deactivate()
         if 'choices' in self.metadata:
-            op = self.metadata['op']
-            data = {
-                'choice': None,
-                'metadata': self.get_metadata(),
-            }
-            if key in self.metadata['choices']:
-                data['choice'] = self.metadata['choices'][key]
-            self.cleanup(op)
-            self.event.emit('command-bar:keypress', data)
+            self.quit({'choice': self.metadata['choices'].get(key)})
```

Comment:
Nice use of the ```get()``` here to clean up the old code a bit! Note to self to use that more often...